test-cases/gutenberg/writing-flow/rich-text-formatting.md

```diff
@@ -94,3 +94,21 @@ Move the cursor around
 - Type some spaces, then write some text, then type some more spaces
 - Select the _text_ and apply one format, for example, bold
 - Check that the leading and trailing spaces aren't removed 
+
+##### TC009
+
+**Test autocorrection doesn't apply formatting to Heading**
+
+- Add a Heading block without any formatting applied.
+- Type a sentence with a word misspelled that will be autocorrected by the editor.
+- Highlight the word after autocorrection.
```

Comment:
Nitpick: I'm wondering if it would be more clear if we specify that the word to highlight is the misspelled one, wdyt?
```suggestion
- Highlight the misspelled word after autocorrection.
```

Reply:
I thought this could be a bit tricky to describe, because at this point it's no longer misspelled and might add a little confusion depending on how someone reads it. 😄

What do you think about something like:

"Highlight the word that was autocorrected."

Reply:
We could even insert a line in between to set the expectation that an autocorrection should've taken place, if that helps.

Reply:
Oh, good point 🤦. Yeah, that wording sounds way better, thanks 🙇 .